### DISCUSSION TITLE
Seminar recordings

### DISCUSSION THREAD

**Discussion by ChristineStawitz-NOAA** [NOAA-FIMS/FIMS#297] [Announcements]
You can find the recording of my Feb 3rd quant sem [here](https://youtube.com/watch?v=iDMO7YjRMuk)

**Comment by kellijohnson-NOAA** [comment]
Below are some additional recorded talks that have been given about FIMS
* [National Stock Assessment Seminar Series](https://www.youtube.com/watch?v=Fx45l7b6SLA) by Christine Stawitz on September 23, 2021
* [National Stock Assessment Seminar Series](https://www.youtube.com/watch?v=5Dhf9zFc4uc) by Andrea Havron on January 4, 2023
* [UW SAFS Quantitative Seminar](https://youtube.com/watch?v=iDMO7YjRMuk) by Christine Stawitz on February 3, 2023
* [Lightning ⚡ talk to NMFS R Users Group](https://www.youtube.com/watch?v=fmXPzEJG8ms) by Andrea Havron, at minute 9:36 of the recording, on Jan 31, 2023
* [National Stock Assessment Seminar Series](https://www.youtube.com/watch?v=tYfyBtxjXk0&t) by Kelli Johnson, on January 3, 2025
* [National Stock Assessment Seminar Series](https://www.youtube.com/watch?v=eCVi3FaL7nE) by Kelli Johnson on April 2, 2026
